SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2665,7 +2665,6 @@ void USpatialNetDriver::TryFinishStartup()
 		}
 		else if (VirtualWorkerTranslator.IsValid() && !VirtualWorkerTranslator->IsReady())
 		{
-			GlobalStateManager->QueryTranslation();
```

Comment:
No longer required as vw translation is part of well known gdk components, which all server workers have interest in. Additionally this query was invalidating the local server's state that had authority over the translation and was manually shorting updates.